### ISSUE TITLE
Why loss function value is too high? Is it expecteted result?

### ISSUE THREAD

**Issue by padeirocarlos** [yhhhli/BRECQ#41]
I tried running your code for with a pre-trained ResNet50 and MobilieNetV2 model. I got loss function value for output and pred losses:

rec_loss = lp_loss(pred, tgt, p=self.p)
:param pred: output from quantized model
:param tgt: output from FP model
[https://github.com/yhhhli/BRECQ/blob/main/quant/block_recon.py#L149](url)

pd_loss = self.pd_loss(F.log_softmax(output / self.T, dim=1), F.softmax(output_fp / self.T, dim=1)) / self.lam
:param pred: output from quantized model
:param tgt: output from FP model
[https://github.com/yhhhli/BRECQ/blob/main/quant/block_recon.py#L151](url)

Are there additional settings I missed?
